test/unit/cinstance_test.rb

```diff
@@ -24,60 +24,60 @@ def teardown
   test 'Cinstance.live_at returns cinstances live at given time' do
     cinstance_one = FactoryBot.create(:cinstance)
 
-    Timecop.travel(1.year.ago)
+    travel_to(1.year.ago)
     cinstance_two = FactoryBot.create(:cinstance)
 
-    Timecop.return
+    travel_back
```

Comment:
use a block

Reply:
You can't because you need `cinstance_two` to be on scope in the assertion